index.bs

```diff
@@ -1385,13 +1371,10 @@ Create a named {{MLOperand}} based on a descriptor, that can be used as an input
     </div>
     1. If |name| is empty, then [=exception/throw=] a {{TypeError}}.
     1. If [=MLOperandDescriptor/checking dimensions=] given |descriptor| returns false, then [=exception/throw=] a {{TypeError}}.
-    1. If any of the following sub-steps fail, [=exception/throw=] an "{{OperationError}}" {{DOMException}}.
+    1. *Make graph connections:*
         1. Let |operand| be the result of [=creating an MLOperand=] given [=this=] and |descriptor|.
         1. Set |operand|.{{MLOperand/[[name]]}} to |name|.
-        1. Make a request to the underlying platform to:
-            1. Create an [=implementation-defined=] platform input operand |operandImpl| given |descriptor|.
-            1. Set |operand|.{{MLOperand/[[operand]]}} to |operandImpl|.
-            1. Register |operand| as an input.
+        1. Register |operand| as an input.
```

Comment:
Heh, I was hoping to defer this. :) But sure - a3cd48f6c704a73941d73d7151edb425aea0a9bb is a first stab at this. Is this what you were thinking?

Reply:
https://github.com/webmachinelearning/webnn/commit/a3cd48f6c704a73941d73d7151edb425aea0a9bb looks good. The only remaining one is computational graph output. Because it is referenced by build method, we probably can defer it to #457.